homeassistant/components/heos/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: done
+    comment: Integration is a local push integration
+  brands: done
+  common-modules: todo
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: todo
+  test-before-setup: done
+  unique-config-entry:
+    status: exempt
+    comment: Only a single config entry is supported.
```

Comment:
If only a single config entry is supported, then it should be set accordingly inside the manifest:
```json
  "requirements": ["pyheos==0.7.2"],
  "single_config_entry": true,
  "ssdp": [
    {
      "st": "urn:schemas-denon-com:device:ACT-Denon:1"
    }
  ]
```

Reply:
Oh neat! I'll set this rule as `todo` and address in a separate change.